From c636a9548aa241f9525979f1bcb3e7a19cf70fb4 Mon Sep 17 00:00:00 2001
From: Sam James <sam@gentoo.org>
Date: Tue, 8 Aug 2023 02:19:57 +0100
Subject: [PATCH 160/180] ld: Fix retain7a.d XFAIL/notarget entry for hppa

PR 30733
* ld/testsuite/ld-elf/retain7a.d: Fix XFAIL entry for hppa to match
  hppa{1.1,2.0}*, like hppa2.0-unknown-linux-gnu which Gentoo uses.

Signed-off-by: Sam James <sam@gentoo.org>
(cherry picked from commit e3b66187192ce6840df283c00f6395bb0ff15cf5)
---
 ld/testsuite/ld-elf/retain7a.d | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ld/testsuite/ld-elf/retain7a.d b/ld/testsuite/ld-elf/retain7a.d
index 314fe0f5733..38d9a9ba255 100644
--- a/ld/testsuite/ld-elf/retain7a.d
+++ b/ld/testsuite/ld-elf/retain7a.d
@@ -2,7 +2,7 @@
 #source: retain7.s
 #ld: -e _start --gc-sections
 # hppa-linux chooses ELFOSABI_GNU regardless of GNU feature use
-#notarget: ![supports_gnu_osabi] ![check_gc_sections_available] hppa-*-linux*
+#notarget: ![supports_gnu_osabi] ![check_gc_sections_available] hppa*-*-linux*
 #readelf: -h
 
 #failif
-- 
2.43.0

